### PR TITLE
Bp cted 1544

### PR DESCRIPTION
**Pull request by BrendonPretorius**
Created "Contact Support" page
Screenshot 2020-11-17 at 12 24 06
Screenshot 2020-11-17 at 12 20 54
Screenshot 2020-11-17 at 12 23 19
Screenshot 2020-11-17 at 12 23 08

### CODE REVIEW

File: classes/output/contact_us.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    }
+
+    /**
+     * Function to get the mappings
+     * @return array
+     */
+    public function get_reasons() {
+        global $DB;
+
+        $results = $DB->get_records('gs_contactus_mappings', null, '', 'id, formreason');
```
Comment (JJSwanevelder) [inline classes/output/contact_us.php:62]: Could potentially return the db result only. Meaning:
`return array_values($DB->get_records('gs_contactus_mappings', null, '', 'id, formreason'))`

**Review by JJSwanevelder — APPROVED** [review]
One small consideration - rest looks good

File: classes/output/contact_us.php (hunk 1 above)
Comment (BrendonPretorius) [reply classes/output/contact_us.php:62]: Yes, you are correct. Sorry, was originally doing the json encode/decode hack but I see mustache can handle arrays of objects fine. Will amend

Comment (JJSwanevelder) [reply classes/output/contact_us.php]: Happy, thank you

**Review by rumanod — APPROVED** [review]

**Pull request merged**